requirements.txt

```diff
@@ -1,8 +1,9 @@
 numpy>=1.7
 pandas>=0.13.1
-mhctools>=1.0.0
+mhctools>=1.3.0
 varcode>=0.3.17
 pylint>=1.4.4
 nose>=1.3.6
 gtfparse>=0.0.4
-mhcnames
\ No newline at end of file
+mhcnames
+pypandoc  # needs pandoc: brew install pandoc
```

Comment:
We don't have consistency on this one yet; we use this in releasing vaxrank and a couple others. Happy to remove this for now, and would like to solve the repo consistency issue separately.